eng/templates/variables/installer-pipelines.yml

```diff
@@ -0,0 +1,10 @@
+# Information about dotnet/installer pipelines that won't change and is useful in multiple pipelines
```

Comment:
What do you see as the scope of these shared variables?  Is it more than just the two release pipelines?  I ask because there appear to be more variables that are shared between the two to declared in each pipeline .e.g blobContainerName

Reply:
These are the shared variables that are needed to run `get-build-info.yml`. Unfortunately you can't define `variables` in templates.

I would not consider the blob containers to be the same. They just happen to have the same name, but they're two separate blob containers in different storage accounts.

I'd be interested to hear any other suggestions you have for organizing variables.

Reply:
Yes you raise a good point about the blob containers not needing to be the same.  I have no great suggestions here.